test/parallel/test-repl.js

```diff
@@ -414,7 +414,10 @@ function error_test() {
       expect: `${prompt_multiline}'foo \\n'\n${prompt_unix}` },
     // Whitespace is not evaluated.
     { client: client_unix, send: ' \t  \n',
-      expect: prompt_unix }
+      expect: prompt_unix },
```

Comment:
@refack No, it's not. That's an error on my part. Will remove it. It's from an unrelated PR I was testing.